src/cryptography/hazmat/bindings/_rust/x509.pyi

```diff
@@ -53,6 +53,12 @@ class ServerVerifier:
     def subject(self) -> x509.verification.Subject: ...
     @property
     def validation_time(self) -> datetime.datetime: ...
+    def verify(
+        self,
+        leaf: x509.Certificate,
+        intermediates: list[x509.Certificate],
+        store: Store,
```

Comment:
Shouldn't store be on the verifier? Seems weird for it not to be.

But maybe I'm missing a use case?

Reply:
I think it could go either way -- I can't remember why we didn't do that earlier, if there was a reason. Happy to move it 🙂 

(I'm guessing the idea here would be `PolicyBuilder().store(...).build_server_verifier(...)`?)